src/backends/plotly.jl

```diff
@@ -448,9 +454,15 @@ const _plotly_markers = KW(
     :hline      => "line-ew",
 )
 
-function plotly_subplot_index(sp::Subplot)
-    spidx = sp[:subplot_index]
-    spidx == 1 ? "" : spidx
+# find indicies of axes to which the supblot links to
+function plotly_link_indicies(plt::Plot, sp::Subplot)
+    if plt[:link] in (:x, :y, :both)
```

Comment:
Plotly can't support `link = :all` with synchronized interactivity should we add a warning about this here?